client/my-sites/site-settings/search.jsx

```diff
@@ -69,22 +70,55 @@ class Search extends Component {
 		);
 	}
 
-	renderSettingsContent() {
-		const { activatingSearchModule, searchModuleActive, translate } = this.props;
+	renderSettingsContent( activatingSearch, searchActive ) {
+		const { translate } = this.props;
 
 		return (
 			<div className="search__module-settings site-settings__child-settings">
-				{ activatingSearchModule && (
+				{ activatingSearch && (
 					<FormSettingExplanation>{ translate( 'Activating search…' ) }</FormSettingExplanation>
 				) }
 
-				{ searchModuleActive && <div>{ this.renderSearchExplanation() }</div> }
+				{ searchActive && <div>{ this.renderSearchExplanation() }</div> }
 			</div>
 		);
 	}
 
-	renderSettings() {
-		const { isRequestingSettings, isSavingSettings, siteId, translate } = this.props;
+	renderWPComSettings() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+			fields,
+			handleAutosavingToggle,
+			translate,
+		} = this.props;
+
+		return (
+			<FormFieldset>
+				{ this.renderInfoLink( 'https://support.wordpress.com/search/' ) }
```

Comment:
This link redirects to https://en.support.wordpress.com/search-engines/, which seems a bit off. Perhaps we should link to the Jetpack Search documentation?